src/packages/pickerview/doc.en-US.md

```
@@ -0,0 +1,94 @@
+
+| Property | Description | Type | Default |
+| --- | --- | --- | --- |
+| label | Text of column | `string` \| `number` | `-` |
+| value | Value of column | `string` \| `number` | `-` |
+| children | Cascader Option | `PickerOptionItem[]` | `-` |
+
+## Theming
+
+### CSS Variables
+
+The component provides the following CSS variables, which can be used to customize styles. Please refer to [ConfigProvider component](#/en-US/component/configprovider).
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)